resources/css/index.css

```
@@ -0,0 +1,186 @@
+@font-face {
+  font-family: "Yeseva One";
+  src: url(../fonts/YesevaOne.otf);
+}
+
+@font-face {
+  font-family: "Lato";
+  src: url(../fonts/Lato.ttf);
+}
+
+@font-face {
+  font-family: "GenB";
+  src: url(../fonts/GenB.ttf);
+}
+
+body, html {
+  margin: 0;
+  padding: 0;
+}
+
+.wrapper {
+  margin: 0 auto;
+  width: 1600px;
+}
+
+.button {
+  box-sizing: border-box;
+  color: #fff;
+  display: inline-block;
+  font-size: 18px;
+  font-family: "Lato", sans-serif;
+  text-transform: uppercase;
+  text-align: center;
+  text-decoration: none;
+  padding: 15px 35px;
+  margin-top: 30px;
+}
+
+.button-book {
+  border: 1px solid black;
+  background-color: rgba(0, 0, 0, 0.6);
+  margin-right: 25px;
+}
+
+.button-menu {
+  border: 1px solid #fff;
+  background-color: rgba(255, 255, 255, 0.2);
+}
+
+.container {
+  background-attachment: fixed;
+  background-image: url(../images/background/Bg.png);
+  background-position: 50% 0px;
+  height: 750px;
+  width: 1600px;
```

You've already defined width
```suggestion
  width: 100%;
```

---

Fixed.